src/twisted/web/_flatten.py

```diff
@@ -167,6 +166,39 @@ def escapedCDATA(data: Union[bytes, str]) -> bytes:
     return data.replace(b"]]>", b"]]]]><![CDATA[>")
 
 
+def _hasMSOComments(html_string: Union[str, bytes]) -> bool:
```

Comment:
even if this is a private method, it needs a docstring.

This is a good place to let other developers know about why this method exists.
Maybe add some more info about how MS Office comments work or a link to the MS Office specs.

Reply:
[4810253](https://github.com/twisted/twisted/pull/11805/commits/481025321d7b95b19ed9213517e821a6a118128b)